t/plugin/jwt-auth.t

```diff
@@ -327,3 +327,118 @@ code: true body: passed
 code: true body: passed
 --- no_error_log
 [error]
+
```

Comment:
your test case style is wrong.

please take a look at this doc.

https://github.com/apache/incubator-apisix/blob/429f0165c202035b18b4437a84c6edc9007fc440/Contributing.md#check-code-style-and-test-case-style